networkx/algorithms/dag.py

```diff
@@ -139,6 +141,14 @@ def topological_sort(G, nbunch=None, reverse=False):
 
     if nbunch is None:
         nbunch = G.nodes_iter()
+        def direct_descendants(w):
+            return G[w]
+    else:
+        nbunch_index = {x: i
+                        for i, x in enumerate(nbunch)}
```

Comment:
I guess @SanketDG meant 
```py
        nbunch_index = {x: i for i, x in enumerate(nbunch)}
```
which is not longer than 79 columns. Maybe that's why split it earlier.